ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -761,9 +673,11 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
           )
     }
 
-    @Explanation("""This error occurs if the Daml engine interpreter can not find a referenced contract. This
-                   |can be caused by either the contract not being known to the participant, or not being known to
-                   |the submitting parties or already being archived.""")
+    @Explanation(
+      """This error occurs if the Daml engine interpreter can not find a referenced contract. This
+        |can be caused by either the contract not being known to the participant, or not being known to
+        |the submitting parties or already being archived."""
+    )
     @Resolution("This error type occurs if there is contention on a contract.")
     object ContractsNotFound
```

Comment:
Should `ContractsNotFound` and `ContractNotFound` be co-located in the code?

Reply:
Actually those two can be unified. Good catch.